### DISCUSSION TITLE
[NFT] Bonfire - A way to burn wood

### DISCUSSION THREAD

**Discussion by Kiulg** [sunflower-land/sunflower-land#1130] [Ideas]
### Bonfire

As we know, wood and stone are the two biggest resources that are accumulated right now. So I propose a simple NFT to burn wood continually (literally)  It may cost:

> ![wood](https://user-images.githubusercontent.com/18270876/175203820-92d8ed79-e585-4f3d-88e1-9dfa26a8d5dc.png) x 40
>
> ![stone](https://user-images.githubusercontent.com/18270876/175203727-2c0021ad-8bab-408a-bb61-6c31c658414e.png) x 40
>
>![token](https://user-images.githubusercontent.com/18270876/175205662-54b7d180-b138-4225-ab03-f774386db75b.png) x 5

With just 1 of 2 possible mechanics and just 1 action. You can put 40 wood and 24 hours after will be available again.

- Drop you a Coal, a new resource that could be used to buy or fabricate future items like cakes, furnaces or whatever.
- Drop you 10.2 SFL as a way to convert wood to SFL in game with a very little reward.

These are the sprites:

Bonfire On
![ingame](https://user-images.githubusercontent.com/18270876/175460773-af6e16a3-c2a4-4265-b30b-5c1cb813fa8a.gif))![bonfire](https://user-images.githubusercontent.com/18270876/175210195-e6874e6a-878c-4ca4-85f0-6720d6876bb9.gif)![bonfire](https://user-images.githubusercontent.com/18270876/175210275-c682413d-83e2-43d4-abc1-39e5e4caa5ef.gif)

Bonfire Off
![ingame2](https://user-images.githubusercontent.com/18270876/175207980-c1863485-a8a6-4877-9c13-d4090ff6776f.png)![bonfire-offb](https://user-images.githubusercontent.com/18270876/175207987-eda06213-9762-4a5c-9761-cb225912cf82.png)![bonfire-off](https://user-images.githubusercontent.com/18270876/175207992-93848de7-47d7-4cb7-afe1-6b89dd84f209.png)

Let me know what do you think :)

**Comment by Demonbane18** [comment]
I'm thinking it could be used to cook some meat like chicken meat, sheep meat, cow meat, etc. so it could be used to also burn animal nfts in the future and also other crops by making them into food recipes and help replenish stamina for the player on the Land expansion update. I think also it could be designed after Genshin Impact's campfire
![image](https://user-images.githubusercontent.com/65271203/175460056-05705d2e-dd22-4bab-ac3e-9445e5930ba4.png)

**Comment by Kiulg** [comment]
Cauldron could be a NFT T2 for everyone, its a nice idea

**Comment by Yougottabekiddingme123** [comment]
Most players don't have an accumulated  stack of wood and stone. In fact a common complaint has been that players are not making enough sfl to mine resources.

On the otherhand there is a small minority who own a t3 beaver and have a large stack of wood. They would be the main beneficiaries  of such a bonfire.

**Reply by SuperSynapse** [reply]
> Most players don't have an accumulated stack of wood and stone. In fact a common complaint has been that players are not making enough sfl to mine resources.
> 
> On the otherhand there is a small minority who own a t3 beaver and have a large stack of wood. They would be the main beneficiaries of such a bonfire.

While I do understand where you're coming from, once you agree NFTs have a value and a benefit this will always be the case.

What I would like to see is a marketplace, lending program, or other group activities that bring people with different NFTs together.

**Reply by Aminbtdds** [reply]
Imo, inflation of wood and stone doesn't matter at current time, with land expansion and woods and stones becoming main resource to expand lands, the use case will appear and burn the accumulated resources.

**Reply by Kiulg** [reply]
My justification is according to this #1099 and uses may add some extra gamification just as say [Demonbane18](https://github.com/Demonbane18)

**Comment by JcEii** [comment]
I love this bonfire art :3 made by boss @Kiulg

**Reply by Kiulg** [reply]
Thanks friend :D

**Comment by ShiningMinions** [comment]
I have a feeling when land expansion comes out, this will have a direct relevant place:

See: https://github.com/sunflower-land/sunflower-land/discussions/1159
